content/en/blog/_posts/2019-09-23-san-diego-contributor-summit

```diff
@@ -0,0 +1,75 @@
+
+**Authors: Paris Pittman (Google), Jeffrey Sica (Red Hat), Jonas Rosland (VMWare)**
+
+[Contributor Summit San Diego 2019 Event Page]
+Registration is now open and in record time, we’ve hit capacity for the
+*new contributor workshop* session of the event! Waitlist is now available.
+
+Sunday, November 17
+Contributor Celebration:
+[QuartYard]
+Address: 1301 Market Street, San Diego, CA 92101
+Time: 6:00PM - 9:00PM
+
+Monday, November 18
+Contributor Summit:
+[Marriott Marquis San Diego Marina]
+Address: 333 W Harbor Dr, San Diego, CA 92101
+Time: 9:00AM - 5:00PM
+
+While the Kubernetes project is only five years old, we’re already going into our
+9th Contributor Summit this November in San Diego before KubeCon + CloudNativeCon.
+The rapid increase is thanks to adding European and Asian Contributor Summits to
+the North American events we’ve done previously. We will continue to run Contributor
+Summits across the globe, as it is important that our contributor base grows in
+all forms of diversity.
+
+Kubernetes has a large distributed remote contributing team, from [individuals and
+organizations] all over the world. The Contributor Summits give the community three
+chances a year to get together, work on community topics, and have hallway track
+time. The upcoming San Diego summit is expected to bring over 450 attendees, and
+will contain multiple tracks with something for everyone. The focus will be around
+contributor growth and sustainability. We're going to stop here with capacity for
+future summits; we want this event to offer value to individuals and the project.
+Getting work done and learning is hard in groups larger than this but it's a priority
+of ours that you have a productive (+fun!) time.
+
+This summit is unique as we’ve taken big moves on sustaining ourselves, the
+contributor experience event team. Taking a page from the release team’s playbook,
+we have added additional core team and shadow roles making it a natural mentoring
+(watching+doing) relationship. The shadows are expected to fill another role at
+one of the three events in 2020, and core team members to take the lead.
+In preparation for this team, we’ve open sourced our [rolebooks, guidelines,
+best practices] and opened up our [meetings] and [project board]. Our team makes up
+many parts of the Kubernetes project and takes care of making sure all voices
+are represented.
+
+Are you at KubeCon + CloudNativeCon but can’t make it to the summit? Check out
+the [SIG Intro and Deep Dive sessions] under the 'maintainer track' label during
+KubeCon + CloudNativeCon to participate in Q&A and hear what’s up with each
+Special interest Group (SIG). We’ll also record all of Contributor Summit’s
+presentation sessions, take notes in discussions, and share it back with you,
+after the event is complete.
+
+We hope to see you all at Kubernetes Contributor Summit San Diego, make sure you
+head over and [register right now]! This event will sell out - here’s your warning.
+:)
```

Comment:
Would a :smile: (in Unicode) work better here?

Reply:
added :grin: